.github/workflows/release.yml

```diff
@@ -32,12 +55,39 @@ concurrency:
   group: ${{ github.workflow }}-${{ github.event.workflow_run.head_branch || inputs.branch }}
 
 jobs:
+  repository-data:
+    runs-on: ubuntu-latest
+    outputs:
+      token: ${{ steps.topics.outputs.token }}
+      topics: ${{ steps.topics.outputs.topics }}
+    steps:
+      - id: token
+        uses: actions/create-github-app-token@5d869da34e18e7287c1daad50e0b8ea0f506ce69 # v1.11.0
+        with:
+          app-id: ${{ secrets.APP_ID }}
```

Comment:
If I run this from the PR it tells me `app-id` is unset, so I'm guessing `secrets.APP_ID` is unset as I'm PRing from a forked PR and that when its merged all will work - I maybe be completely wrong on this tho!

Reply:
Confirmed offline that I can't access secrets here as I'm PR from a fork. Not sure how we can 100% test this end to end without merging. But in the scheme of things, if it breaks once its on master we can just fix it. I tested it quite a bit using `github.token` instead.